core/src/test/scala/com.codacy.analysis.core/git/GitSpec.scala

```diff
@@ -19,6 +20,15 @@ class GitSpec extends Specification with NoLanguageFeatures {
       }).get
     }
 
+    "get the current commit uuid" in {
+      withTemporaryGitRepo((directory, _, _, _) => {
+        val expectedUuid = Process(Seq("git", "rev-parse", "HEAD"), directory.toJava).!!.trim
```

Comment:
We should never depend on commands to do tests. The test should have a fixed UUID.